[ews] Rename CI builds to Apple Internal builds where appropriate

aj062 opened the pull request:
#### 209de5fb7be32410c845dcb7e73b917610cbea82
<pre>
[ews] Rename CI builds to Apple Internal builds where appropriate
<a href="https://bugs.webkit.org/show_bug.cgi?id=296586">https://bugs.webkit.org/show_bug.cgi?id=296586</a>
<a href="https://rdar.apple.com/156939761">rdar://156939761</a>

Reviewed by Brianna Fan.

* Tools/CISupport/ews-app/ews/common/github.py:
(GitHubEWS):
(GitHubEWS.generate_comment_text_for_change):
(GitHubEWS.should_include_apple_internal_builds):
(GitHubEWS.add_or_update_comment_for_change_id):

Canonical link: <a href="https://commits.webkit.org/297978@main">https://commits.webkit.org/297978@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/138328832f1eafc4d5793e157ce3e57a39616311

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/113599 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/131/builds/33306 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/138/builds/23746 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/119770 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/64367 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | ⏳ 🛠 ios-apple 
| [  ~~🧪 bindings~~](https://ews-build.webkit.org/#/builders/9/builds/115488 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/130/builds/33913 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/123/builds/41876 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/119770 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/64367 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | ⏳ 🛠 mac-apple 
| [  ~~🧪 webkitperl~~](https://ews-build.webkit.org/#/builders/11/builds/116547 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/130/builds/33913 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/102096 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/119770 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/130/builds/33913 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/20217 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/63496 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/130/builds/33913 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/20302 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/123005 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [  ~~🛠 vision~~](https://ews-build.webkit.org/#/builders/128/builds/40602 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/123/builds/41876 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/123005 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [  ~~🛠 vision-sim~~](https://ews-build.webkit.org/#/builders/121/builds/40993 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/98307 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/123005 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/17939 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/36839 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18246 "Built successfully and passed tests") | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/127/builds/40485 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/40144 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/43456 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/41901 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | | | 
<!--EWS-Status-Bubble-End-->

emw-apple commented on Tools/CISupport/ews-app/ews/common/github.py:
```diff
@@ -272,11 +272,11 @@ def generate_comment_text_for_change(self, change, include_ci_builds=False):
 
-    def should_include_ci_builds(self, pr_author, pr_project):
+    def should_include_apple_internal_builds(self, pr_author, pr_project):
         return (pr_author in self.approved_user_list_for_cibuilds) and (pr_project in GITHUB_PROJECTS)
```
Do you also want to rename `approved_user_list_for_cibuilds`?

aj062 replied:
I was planning to rename that in another change I am working on which touches that. But I just updated this PR to rename `approved_user_list_for_cibuilds` as well.

briannafan commented on Tools/CISupport/ews-app/ews/common/github.py:
```diff
@@ -272,11 +272,11 @@ def generate_comment_text_for_change(self, change, include_ci_builds=False):
     def github_status_for_queue(self, change, queue):
-        if queue in self.APPLE_QUEUES:
+        if queue in self.APPLE_INTERNAL_QUEUES:
             return self.github_status_for_cibuild(change, queue)
```
Should we also rename `self.github_status_for_cibuild`?

aj062 replied:
I thought about it but skipped renaming it because this part is still kind-of generic. CI Build is a generic table which can accept data from any build system, apple internal build being just one of those. Currently there is nothing in `github_status_for_cibuild` method which is tightly coupled with Apple Internal builds.

aj062 committed:
[ews] Rename CI builds to Apple Internal builds where appropriate
https://bugs.webkit.org/show_bug.cgi?id=296586
rdar://156939761

Reviewed by Brianna Fan.

* Tools/CISupport/ews-app/ews/common/github.py:
(GitHubEWS):
(GitHubEWS.generate_comment_text_for_change):
(GitHubEWS.should_include_apple_internal_builds):
(GitHubEWS.add_or_update_comment_for_change_id):

Canonical link: https://commits.webkit.org/297978@main
